classes/class-wc-connect-help-provider.php

```diff
@@ -416,13 +517,15 @@ public function page() {
 			$form_schema = new stdClass();
 			$form_schema->properties = new stdClass();
 
+			$path = "/wc/v1/connect/self-help";
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/376/commits/77e6c33e877d6931f17e4958e855171bc1cf814b